assets/css/code.css

```diff
@@ -2,46 +2,102 @@ code {
   color: #3f76a3 !important;
   background: #eee;
 
-  font-family: Nova Mono, Iosevka, Ubuntu Mono, Monaco, Consolas, monospace !important;
-
-  padding: 1px;
-  white-space: nowrap;
+  padding: 2px;
 
   transition: .3s background;
 }
 
-code a:hover {
-  background: unset;
-}
-
-code var {
-  color: #db5375;
-  font-style: normal;
-}
-
-code var.number {
-  color: #8093f1;
-}
-
-code var.string {
-  color: #688b58;
+.highlighter-rouge pre.highlight {
+  margin-bottom: 1.5rem;
+  padding: 1.5rem;
+  white-space: pre-wrap;
```

Comment:
I think for code it makes sense to not wrap long lines but having the block add a vertical scrollbar if the line gets too long.

Reply:
Yeah, it increases the screen width for me as well.

I managed to "fix" it by removing the `d-flex` class from the `div.container` directly inside the `main` tag. flex still confuses me a bit, so I'm not sure if this will have any side effects somewhere else.

Reply:
Curious, I'm not sure why it even affects it in such a way at the moment, I'll try figuring it out tomorrow. That whole flexbox mess is necessary to properly stretch everything to screen height so that the latest posts thing is placed at the bottom; there might be a better way to solve that, but again, I'm currently not sure what that would be.

Reply:
I'm sure you'll find a good solution :)

Reply:
Man, good thing I stumbled upon [this](https://weblog.west-wind.com/posts/2016/Feb/15/Flexbox-Containers-PRE-tags-and-managing-Overflow), doubt I would've figured it out on my own. What a nasty detail.